src/screens/DiscoveryScreenV2/elements/DiscoveryTitleSeparator.js

```diff
@@ -1,35 +1,78 @@
 import * as React from 'react';
-import {Text} from 'react-native';
-
+import {Animated, Text, View} from 'react-native';
+import ArrowDown from '../../../../assets/icons/arrow-down.svg';
 import {fonts} from '../../../utils/fonts';
 import {COLORS} from '../../../utils/theme';
 
 /**
  * @typedef {Object} DiscoveryTitleSeparatorProp
  * @property {String} text
+ * @property {Boolean} showArrow
+ * @property {Boolean} [rotateArrow]
  */
 /**
  *
- * @param {DiscoveryTitleSeparatorProp} prop
+ * @param {DiscoveryTitleSeparatorProp} props
  */
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The JSDoc comments are well written and provide a clear understanding of the component's props. Good job!



<!-- This is an auto-generated comment by OSS CodeRabbit -->